React-Native/TodoList/src/app/screens/BookDetail/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { View, Text } from 'react-native';
+
+class BookDetail extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const { params } = navigation.state;
+
+    return {
+      title: params ? params.title : 'A Nested Details Screen'
+    };
+  };
+
+  render() {
+    const { navigation } = this.props;
+    return (
+      <View>
+        <Text>{'Title: '}{navigation.getParam('title', 'Non title')}</Text>
```

Comment:
ok.. is not wrong but i think you can pass only one parameter with an object. Inside you can have all data you are showing. Then.. to render the data (let's suppose you use a param called item passed in navigation) you have to use literals: `Title: ${item.title}`. Then.. the placeholder, let's try this: do not render the Text that has no data.